articles/search/cognitive-search-skill-image-analysis.md

```diff
@@ -47,56 +47,270 @@ Parameters are case-sensitive.
+                                    "name": "name",
+                                    "type": "Edm.String",
+                                    "searchable": true,
+                                    "filterable": false,
+                                    "facetable": false
+                                },
+                                {
+                                    "name": "faceBoundingBox",
+                                    "type": "Collection(Edm.ComplexType)",
+                                    "fields": [
+                                        {
+                                            "name": "x",
+                                            "type": "Edm.Int32",
+                                            "searchable": false,
+                                            "filterable": false,
+                                            "facetable": false
+                                        },
+                                        {
+                                            "name": "y",
+                                            "type": "Edm.Int32",
+                                            "searchable": false,
+                                            "filterable": false,
+                                            "facetable": false
+                                        }
+                                    ]
+                                },
+                                {
+                                    "name": "confidence",
+                                    "type": "Edm.Double",
+                                    "searchable": false,
+                                    "filterable": false,
+                                    "facetable": false
+                                }
+                            ]
+                        },
+                        {
+                            "name": "landmarks",
+                            "type": "Collection(Edm.ComplexType)",
+                            "fields": [
+                                {
+                                    "name": "name",
+                                    "type": "Edm.String",
+                                    "searchable": true,
+                                    "filterable": false,
+                                    "facetable": false
+                                },
+                                {
+                                    "name": "confidence",
+                                    "type": "Edm.Double",
+                                    "searchable": false,
+                                    "filterable": false,
+                                    "facetable": false
+                                }
+                            ]
+                        }
+                    ]
+                }
+            ]
         },
         {
-            "name": "imageType",
-            "targetName": "myImageType"
+            "name": "description",
```

Comment:
Could you rename this to "descriptions" since it is really a collection of descriptions.

Reply:
I could, but only so far, the API gives back Description, not Descriptions as a field, so that can be mapped into Descriptions at some point. The issue Search is done based on documents, each document can be one image or it can be multiple images (or instance when the source is a multipage PDF with just images in it) this means that the pipeline runs against 1 or multiple images and thus does multiple calls to the API, and since each call gives back 1 description (hence Description), the search pipeline combines those Descriptions for further processing downstream, so at the API level, it is considered 1:1 while at the Search level it can be 1:many.

Reply:
Ok, don't worry about it... I am signing off on the submission. Thanks again! I really appreciate it when folks like you contribute to the community.